src/veupath/redmine/client/genome.py

```diff
@@ -208,6 +203,7 @@ def _get_gff(self) -> None:
             self.gff = gff_path
 
     def _get_replacement(self) -> None:
+        """Set `is_replacement` to true if the genome is expected to be a replacement."""
```

Comment:
```suggestion
        """Set `is_replacement` to True if the genome is expected to be a replacement."""
```
I tend to like to set boolean values in capital case to follow Python coding in the docstrings, I find these more readable.